### ISSUE TITLE
In the new version 3.2.0 the prefixes disappear?

### ISSUE THREAD

**Issue by guillermohaad** [opensourcepos/opensourcepos#1601]
Just leave POS prefix and in previous versions existed RECV, DEV , QUOTA.. ?

**Comment by daN4cat** [comment]
I don't get, what is the issue and how can we see it?

@SteveIreland does that ring a bell?

**Comment by SteveIreland** [comment]
There's not enough information in the request for me to understand it.  I'm assuming "previous versions" means either a version earlier than 3.0 or a version containing a custom change. I see RECV in the code but I haven't started doing any work in this area yet, so I don't know if it's being used.  I will be diving in to this soon so I'll keep any eye out for anything that doesn't look right. 

I have no idea what DEVis used to indicate.  I'm assuming QUOTA translates to QUOTE.  But I've not touched the prefix at the bottom of the receipt, but maybe I should?   I think I saw some embedded code related to the concatenation of POS and the sale_id so I've avoided touching it.
